TicTacToeTest.py

```diff
@@ -0,0 +1,216 @@
+import sys, pygame
+import cv2
+import numpy as np
+import random
+import time
+from time import time
+
+cap = cv2.VideoCapture(0)
+
+# take first frame of the video
+ret, frame = cap.read()
+
+# define range of green color in HSV
+lower_blue = np.array([50,75,50])
+upper_blue = np.array([75,150,200])
+
+# setup initial location of window
+r,h,c,w = 200,120,260,120  # simply hardcoded the values
+track_window = (c,r,w,h)
+
+# set up the ROI for tracking
+roi = frame[r:r+h, c:c+w]
+hsv_roi =  cv2.cvtColor(roi, cv2.COLOR_BGR2HSV)
+
+# Threshold the HSV image to get only blue colors
+mask = cv2.inRange(hsv_roi, lower_blue, upper_blue)
+
+roi_hist = cv2.calcHist([hsv_roi],[0],mask,[180],[0,180])
+cv2.normalize(roi_hist,roi_hist,0,255,cv2.NORM_MINMAX)
+
+# Setup the termination criteria, either 10 iteration or move by atleast 1 pt
+term_crit = ( cv2.TERM_CRITERIA_EPS | cv2.TERM_CRITERIA_COUNT, 10, 1 )
+
+pygame.init()
```

Comment:
In terms of code formatting/layout, to make your code easier to read, wrapping all your setup code into a function could be beneficial.